assets/js/product.js

```diff
@@ -15,6 +15,7 @@ storefrontApp.controller('productController', ['$rootScope', '$scope', '$window'
         $scope.addToWishlistDisabled = false;
         $scope.availableLists = null;
         $scope.listType = null;
+        $scope.averageRating = null;
```

Comment:
averageRating: property not used anywhere?

Reply:
removed